### DISCUSSION TITLE
Gapless

### DISCUSSION THREAD

**Discussion by manio** [hyprwm/Hyprland#4123] [Q&A] [answered]
Hi,
I am using a gapless config:
```
[...]
general {
    gaps_in = 0
    gaps_out = 0
    border_size = 1
[...]
```
But in some circumstances (eg. two `foot` terminals horizontally) I can see that the bottom window is not filled entirely (a background is visible at the bottom edge), like this:
![image](https://github.com/hyprwm/Hyprland/assets/583157/4f0dad0a-eb57-4d50-8167-885f62008f38)
Is this a known problem?
Can I adjust a config to fix this somehow?

**Comment by vaxerski** [comment, marked as answer]
what's your version? IIRC this was fixed a while back, and was especially an issue with scaling.

**Reply by manio** [reply]
You was indeed right, sorry for this.
After upgrade:
`upgraded hyprland (0.30.0-1 -> 0.33.1-2)`
The problem is gone. Thanks!
